medcat/config.py

```diff
@@ -10,8 +10,9 @@ def weighted_average(step, factor):
 
 
 def workers(workers_override=None):
-    return max(cpu_count() - 1, 1) if workers_override is None else workers_override
-
+    # FIXME
```

Comment:
there's a jira for this?

Reply:
Yes, the ongoing CAT-32 should have covered that. Will make some comments.